frontend/src/AboutPageHelper.js

```diff
@@ -3,9 +3,12 @@ import NutritionTable from "./NutritionTable";
 import Reviews from "./Reviews";
 import { Link } from "react-router-dom";
 
+import "./About.css";
+
 function AboutPageHelper(props) {
     const { food, setFood } = props;
     let animalProductsMessage;
+
```

Comment:
Random Line added here, is this a prettier change?

Reply:
That was just to separate between the variable declarations and the rest of the code